### PR TITLE
CA-1159 Added Swagger URL and instructions to README

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary. 
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,28 @@
...
+## Authorization 
+For endpoints that require authorization, you will need a Google Access Token that you can generate using the 
+[gcloud](https://cloud.google.com/sdk/gcloud) command:
+
+`gcloud auth print-access-token`
+
+When you "authorize" in the Bond Swagger UI:
+
+1. Click the `Authorize` button at the top right of the Swagger UI
+1. Copy/paste the generated Access Token from the gcloud CLI into `Value` field
```
Comment (s-rubenstein) [inline README.md:22]: These each have `1.`? Should it be `1.` `2.` and `3.`?

**Review by s-rubenstein — APPROVED** [review]
LGTM other that the one comment.

**Review by andy7i — APPROVED** [review]
This is really helpful! Would it be worth adding the dev bond link as well? (https://broad-bond-dev.appspot.com/api/docs/) Maybe in `## Deploy to the "dev" environment`? Otherwise, 👍  from me!

File: README.md (hunk 1 above)
Comment (dvoet) [reply README.md:22]: that's markdown... the display of markdown will correctly be 1, 2, 3

**Review by dvoet — APPROVED** [review]

**Pull request merged**